package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "lfpwebapp",
+  "version": "0.0.1",
+  "description": "This is a skeleton for a node.js + express.ts app.",
+  "main": "build/app.js",
+  "scripts": {
+    "start": "npm run build && node build/app.js",
+    "build": "node-sass -r src/public/sass -o src/public/css --include-path node_modules/ && gulp",
```

Comment:
Since you engaged `gulp` whole SCSS transpiration can be performed in `gulp` as well. I suggest we engage `gulp` to that task as well to minimise surprise.